cmd/upgrade.go

```diff
@@ -52,7 +53,9 @@ type upgradeCmd struct {
 }
 
 func newUpgradeCmd() *cobra.Command {
-	uc := upgradeCmd{}
+	uc := upgradeCmd{
```

Comment:
This follows the same pattern as in deploy and abstract away the auth and client during tests.